xdsl/dialects/builtin.py

```diff
@@ -1090,6 +1090,22 @@ def verify(self):
                 f"equal to number of dimensions {num_dims}."
             )
 
+    @classmethod
```

Comment:
Should we not have arguments for the other parameters that default to `AnyAttr`?

Reply:
My bad, should be good now!